distribution/lib/Standard/Base/0.0.0-dev/src/Warning.enso

```diff
@@ -188,3 +189,37 @@ merge_matched_warnings value matcher merger =
         new_warnings = merger (result.second.map .value)
         new_warnings.fold result.first acc-> warning->
             Warning.attach warning acc
+
+## PRIVATE
+
+type Prim_Warning
+
+    type Prim_Warning
```

Comment:
```suggestion
    ## PRIVATE
    type Prim_Warning
```

Also the tag should be replicated here. Especially as IIRC, this is the constructor that will be exposed to the searcher and not the one above, so above the comment is ok but (currently) unnecessary, but here it is required to avoid showing this constructor in the searcher.

_Unless_ something changed in the meantime, I'm not 100% up-to-date on this.

Reply:
:+1: